### ISSUE TITLE
Revisiting Replay and Gradient Alignment for Continual Pre-Training of Large Language Models

### ISSUE THREAD

**Issue by istabrak-abbes** [zzz47zzz/awesome-lifelong-learning-methods-for-llm#2]
hello,
I hope you are doing well, 
This is my recent paper accepted to CoLLAs : https://arxiv.org/pdf/2508.01908
Can you concider adding it to the list of papers ?

Thanks

**Comment by zzz47zzz** [comment]
Sure, we will add the paper in the  next update！

---- Replied Message ----
| From | Istabrak ***@***.***> |
| Date | 08/07/2025 21:19 |
| To | zzz47zzz/awesome-lifelong-learning-methods-for-llm ***@***.***> |
| Cc | Subscribed ***@***.***> |
| Subject | [zzz47zzz/awesome-lifelong-learning-methods-for-llm] Revisiting Replay and Gradient Alignment for Continual Pre-Training of Large Language Models (Issue #2) |
istabrak-abbes created an issue (zzz47zzz/awesome-lifelong-learning-methods-for-llm#2)

hello,
I hope you are doing well,
This is my recent paper accepted to CoLLAs : https://arxiv.org/pdf/2508.01908
Can you concider adding it to the list of papers ?

Thanks
